cmake/apple/readme.txt

```diff
@@ -5,12 +5,16 @@ Solarus is licensed under GNU General Public License v3.
 This OSX package contains:
 - Solarus (Solarus.app): A GUI that allows you to run a Solarus quest and
   choose your audio and video settings.
-- Solarus-run (Solarus-run.app): An executable file that directly runs
-  a Solarus quest.
-  Run it with the path of your quest as argument
-  (the directory containing data/ or data.solarus or data.solarus.zip).
+- Solarus-run (Solarus-run.app): A package you can use either to distribute a 
+  standalone quest on other OSX systems with non-developer environment, or to 
+  run the solarus-run binary through the console.
+  The bundle is made to run any quest located into Solarus-run.app/Contents/Resources,
+  just replace the sample quest by your own one.
+  To use solarus-run through the command-line, change the current directory to 
+  "Solarus-run.app/Contents/Resources/" then run it with the path of
+  your quest as argument (the directory containing data/ or data.solarus).
   Without this argument, the current directory is used as quest path.
-  Try also 'solarus -help' to show information about the command-line
+  Try also 'solarus-run -help' to show information about the command-line
```

Comment:
I don't understand. Does it run the quest in Solarus-run.app/Contents/Resources or the quest in the current directory?
This is very confused. It should simply run the quest in the current directory of in the command line, as on all systems.